README.md

```diff
@@ -729,6 +729,7 @@ of these utilities was in the `react-testing-library`.
 Implementations include:
 
 - [`react-testing-library`](https://github.com/kentcdodds/react-testing-library)
+- [`pptr-testing-library`](https://github.com/patrickhulce/pptr-testing-library)
```

Comment:
vue-testing-library isn't mentioned here even though it is elsewhere in the doc, should it be? Is this not the spot for related projects?

Reply:
Definitely should be mentioned!